### ISSUE TITLE
UI only loads every other time

### ISSUE THREAD

**Issue by cnolson** [CitiesSkylinesMods/TMPE#877]
### Describe the problem
Since Sunset Harbor, when I click the TM:PE button in-game, the UI will only load every other time I run the game. The first time I run, I click the button, the button turns blue, but no UI box appears. If I quit and relaunch the game, the UI displays as usual.

Since downloading Sunset Harbor, I have unsubscribed all mods, uninstalled the game, reinstalled, and subscribed to fewer mods, but I get the same result. It happens regardless of the save game I open.

### Steps to reproduce

1. Open game, open save
2. Click TM:PE button. No UI
3. Quit, relaunch, reload
4. Click TM:PE button, opens as usual.

### Log files

[Player.log](https://github.com/CitiesSkylinesMods/TMPE/files/4562196/Player.log)

### Savegame?

https://steamcommunity.com/sharedfiles/filedetails/?id=2079899613

**Comment by krzychu124** [comment]
Could you post **TMPE.log** from game when UI doesn't load?

**Comment by cnolson** [comment]
The log I posted was taken immediately after I opened the game and did not
have the UI load.

**Comment by krzychu124** [comment]
Ok, but we need also a log from the mod which is named TMPE.log (https://github.com/CitiesSkylinesMods/TMPE/wiki/Share-your-Cities-Skylines-log-file)

**Comment by cnolson** [comment]
My apologies, I misread your email.

[TMPE.log](https://github.com/CitiesSkylinesMods/TMPE/files/4562392/TMPE.log)

**Comment by cnolson** [comment]
I had been playing the game, quit the game. quit Steam. relaunched Steam, and relaunched the game. The issue occurred again, and that is the log file I have sent.

**Comment by krzychu124** [comment]
Ok, no errors but I suspect menu is somehow out of your screen. 
Try resetting global config - when you load saved game go to mod options Maintenance tab and click 'reset global config'. That should revert position of menu to default values

**Comment by Xyrhenix** [comment]
```
NullReferenceException: Object reference not set to an instance of an object
  at TrafficManager.LoadingExtension.OnReleased () [0x00000] in <filename unknown>:0 
  at LoadingWrapper.OnLoadingExtensionsReleased () [0x00000] in <filename unknown>:0 
UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:LogException(Exception)
LoadingWrapper:OnLoadingExtensionsReleased()
LoadingWrapper:Release()
LoadingManager:ReleaseRelay()
LoadingManager:OnDestroy()
```

Not quite sure on the no-errors thing, though it looks unrelated. This from the Player.log.

**Comment by kianzarrin** [comment]
Several useful this to test:
- Can you reproduce this problem if UNSUBSCRIBE from all mods but TMPE in a NEW GAME (reset global config and exit to desktop AFTER you unsubscribed from other mods).
- if no can you reproduce this problem if ONLY TMPE is enabled in your saved game?

**Comment by cnolson** [comment]
> Ok, no errors but I suspect menu is somehow out of your screen.
> Try resetting global config - when you load saved game go to mod options Maintenance tab and click 'reset global config'. That should revert position of menu to default values

Resetting the global config brought the menu back! I will try the other options if it stops solving the issue in the future. Thank you for your help.

**Comment by aubergine10** [comment]
That sounds like an v11.4.x issue that we hope to have fixed in v11.5.0.

Related steam discussion with similar issue: https://steamcommunity.com/workshop/filedetails/discussion/1637663252/3581993464541317414/

Tagging PRs #849 and #868

**Comment by krzychu124** [comment]
Yeah I thought about that at first but he is using 11.3.2 Stable, which doesn't have new menu (or maybe it does?)
